### PR TITLE
feat: added navigation and slider

### PR DESCRIPTION
**Pull request by Priya-Patel31**
Navigation and slider added

### CODE REVIEW

**Review by Raghav888 — APPROVED** [review]
Approved

File: component/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    padding: 0 0.5rem;
+    margin: 0;
+    border: none;
+    background-color: transparent;
+  }
+  .search-bar {
+    height: 2rem;
+    background-color: transparent;
+    border: none;
+    outline: 0px;
```
Comment (JantuDeb) [inline component/navigation/navigation.css]: Do not remove outline style.

File: component/slider/slider.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
+.slider-container {
+    width: 50%;
+  }
+  .slider {
+    padding: 0 !important;
```
Comment (JantuDeb) [inline component/slider/slider.css:5]: Avoid using `!important`

**Review by JantuDeb — APPROVED** [review]
I added some suggestions.

File: component/navigation/navigation.css (hunk 1 above)
Comment (Priya-Patel31) [reply component/navigation/navigation.css]: changed it !!

File: component/slider/slider.css (hunk 2 above)
Comment (Priya-Patel31) [reply component/slider/slider.css:5]: but it was necessary that is why I have used .

**Pull request merged**